package.json

```diff
@@ -9,23 +9,31 @@
     "dev": "ts-node-dev ./src/index.ts",
     "db-push": "npx drizzle-kit push:mysql",
     "migrate": "drizzle-kit generate:mysql --schema src/db/schema.ts && ts-node src/db/migrate.ts",
-    "push-commands": "npx tsc && node ./dist/command/pushScript.js"
+    "push-commands": "npx tsc && node ./dist/command/pushScript.js",
+    "build-styles": "tailwindcss -i src/web/index.css -o ./public/style.css",
```

Comment:
Using [TailwindCSS](https://tailwindcss.com/) for styles.
Keep a terminal open running `yarn watch-styles` if you are working on web stuff.